### PR TITLE
Colour Updates

### PR DESCRIPTION
**Pull request by psnsVGC**
Dark Mode message-error edit: http://prntscr.com/9vqw0i
Dark Mode popupmenus edits: http://prntscr.com/9vr3fj and http://prntscr.com/9vrh76 
Normal Mode popupmenus edits: http://prntscr.com/9vrf08 and http://prntscr.com/9vr9ys

### CODE REVIEW

File: style/client.css [hunk 1]
```diff
@@ -135,9 +135,8 @@ button:disabled {
 	background: #111111;
 }
 .dark .message-error {
-	color: #FF5333;
+	color: #BBOBOB;
```
Comment (Rhythms88) [inline style/client.css]: `#BB0B0B`?

Comment (psnsVGC) [inline style/client.css]: Oops big mistake on my part sorry It's 03:15 here

**Comment by psnsVGC** [comment]
making a new pull

**Comment by panpawn** [comment]
You should probably be much more specific in your commit name and commit summary.

**Pull request closed without merging**